packages/framework/fluid-framework/api-report/fluid-framework.alpha.api.md

```diff
@@ -1141,11 +1141,11 @@ export class SchemaFactory<out TScope extends string | undefined = string | unde
     readonly boolean: LeafSchema<"boolean", boolean>;
     static readonly boolean: LeafSchema<"boolean", boolean>;
     protected getStructuralType(fullName: string, types: TreeNodeSchema | readonly TreeNodeSchema[], builder: () => TreeNodeSchema): TreeNodeSchema;
-    readonly handle: LeafSchema<"handle", IFluidHandle<unknown>>;
-    static readonly handle: LeafSchema<"handle", IFluidHandle<unknown>>;
+    readonly handle: LeafSchema<"handle", IFluidHandle_2<unknown>>;
```

Comment:
Have you tried re-adding the necessary imports to avoid this aliasing?

Reply:
I added them in the tree package. TO make these go away I think we would need to duplicate that in the fluid-framework package. That package as its TS config make such unused imports a hard error, so suppressing the lint is not enough.